src/Contracts/View.php

```diff
@@ -109,13 +113,12 @@ public function validate(): bool {
 	 * @return void
 	 */
 	public function process() {
-		return;
 	}
 
 	/**
 	 * Returns the nonce action
 	 *
-	 * @return void
+	 * @return string the nonce action
 	 */
 	public function nonce_action() {
```

Comment:
```suggestion
	public function nonce_action(): string {
```

A return type should also always be set after for a function.